src/main/java/sam/command/FindCommand.java

```diff
@@ -0,0 +1,27 @@
+package sam.command;
+
+import java.util.List;
+
+import sam.Ui;
+import sam.storage.Storage;
+import sam.task.TaskList;
+
+/**
+ * Represents a user command to find a task.
+ */
+public class FindCommand extends Command {
+    public FindCommand(String args) {
+        super(args);
+    }
+
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
+        List<String> list = tasks.findTasks(args);

```

Comment:
Just a minor suggestion to use lists as the variable name to represent collections instead?